plugins/modules/keycloak_client.py

```diff
@@ -581,6 +582,14 @@
                     - For OpenID-Connect clients, client certificate for validating JWT issued by
                       client and signed by its key, base64-encoded.
 
+            x509.subjectdn:
+                description:
+                    - For OpenID-Connect clients, subject which will be used to authenticate the client.
+
+            x509.allow.regex.pattern.comparison:
+                description:
+                    - For OpenID-Connect clients, boolean specifying whether to allow C(x509.subjectdn) as regular expression.
```

Comment:
Same as above. Likely `type: bool` if this is a boolean.
```suggestion
                    - For OpenID-Connect clients, boolean specifying whether to allow C(x509.subjectdn) as regular expression.
                type: ...
                version_added: 9.5.0
```